experimentation/src/main/java/com/automattic/android/experimentation/ExPlat.kt

```diff
@@ -74,20 +81,28 @@ class ExPlat(
     }
 
     private fun getAssignments(refreshStrategy: RefreshStrategy): Assignments {
-        val cachedAssignments = experimentStore.getCachedAssignments() ?: Assignments()
-        if (refreshStrategy == ALWAYS || (refreshStrategy == IF_STALE && cachedAssignments.isStale())) {
+        val cachedAssignments: Assignments =
+            runBlocking { repository.getCached() } ?: Assignments(emptyMap(), 0, 0)
+        if (refreshStrategy == ALWAYS || (
+                refreshStrategy == IF_STALE && assignmentsValidator.isStale(
```

Comment:
Minor (🔍): Consider making this a one-liner, IMHO this reads a bit better: `refreshStrategy == IF_STALE && assignmentsValidator.isStale(cachedAssignments))`